File: HackerRank Solutions/10 Days of Statistics/Day 6 The Central Limit Theorem I.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
import math

x = int(input())
n = int(input())
mu = int(input())
sigma = int(input())
mu1=  mu * n 
sigma1 = sigma * math.sqrt(n)
def cdf(x, mu, sigma):
    Z = (x - mu)/sigma
    return 0.5 * (1 + math.erf(Z / (math.sqrt(2))))
print(round(cdf(x, mu1, sigma1), 4))

File: HackerRank Solutions/10 Days of Statistics/Day 6 The Central Limit Theorem I.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
#include <iomanip>
using namespace std;

double normal_cdf(double x, double mu, double sigma){
    double z = (x - mu)/(sigma*sqrt(2));
    return  (1 + erf(z))/2;  
}
int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    double x = 9800.0;
    double mu = 205.0, sigma = 15.0;
    int n = 49;
    double mu1 = mu * n;
    double sigma1 = sigma * sqrt(n);
    cout << fixed << setprecision(4) << normal_cdf(x, mu1, sigma1);
    return 0;
}